library/junos_install_config

```diff
@@ -150,6 +150,14 @@ options:
         required: false
         default: None
         choices: [1 - 4]
+    commit_onwarnings:
```

Comment:
I think the name of the argument is a misnomer. It doesn't commit on all warnings, just "statement not found" warnings. Therefore, I suggest we change the name of the argument to `ignore_statement_not_found`.